src/mono/mono/mini/aot-compiler.c

```diff
@@ -8804,8 +8804,8 @@ mono_aot_parse_options (const char *aot_options, MonoAotOptions *opts)
 			opts->llvm_outfile = g_strdup (arg + strlen ("llvm-outfile="));
 		} else if (str_begins_with (arg, "export-symbols-outfile=")) {
 			opts->export_symbols_outfile = g_strdup (arg + strlen ("export-symbols-outfile="));
-		} else if (str_begins_with (arg, "compiled-methods-outfile=")) {
```

Comment:
do we know whether anyone relies on the old name for this switch? should we support both the old and new names?

Reply:
No, I don't expect anyone relies on the old name. This is a new feature added this release.